src/alire/alire-toml_index.ads

```diff
@@ -321,7 +321,8 @@ private
    --  false and produce an error message.
 
    procedure Decode_TOML_Package_As_Releases
-     (Pkg         : Package_Type;
+     (Package_Dir : String;
+      Pkg         : Package_Type;
       Environment : Environment_Variables;
       Releases    : out Containers.Release_Sets.Set);
    --  Generate one fully populated release per version loaded in Pkg
```

Comment:
Could you complete the documentation to describe what value callers must provide for `Package_Dir`?

Reply:
Done in new force-push